record/index.html.in

```diff
@@ -88,7 +88,7 @@
                 </div>
                 <span id="rt-loading" ng-show="loading">Loading...</span>
             </div>
-            <footer></footer>
+            <footer ng-show="!loading && displayReady"></footer>
```

Comment:
Does it really have to be based on loading too? We only didn't wanted to show the empty box at the top of the page. We're only looking at the `displayReady` attribute to apply the height logic, so just having `displayReady` should be sufficient.